### ISSUE TITLE
Allow multiple window borders

### ISSUE THREAD

**Issue by dgsasha** [yilozt/rounded-window-corners#105]
Libadwaita draws two window borders, a 1px semi-transparent black border (it's actually done with a box shadow) on the outside of the window (mostly noticeable with the light theme) and a 1px semi-transparent white border inside the window (mostly noticeable with the dark theme). If possible I think this extension should have an option to have two borders, one inside the window, and one on the outside. This may also help with the issue of apps that don't follow the system light/dark theme preference discussed here https://github.com/yilozt/rounded-window-corners/issues/97

**Comment by stpnwf** [comment]
Yes, please, this would be heaven sent, as this extension removes the window borders from my GTK3 theme ([Colloid](https://github.com/vinceliuice/Colloid-gtk-theme)). So it would be nice to be able to add them back.
